app/components/tables/default.js

```diff
@@ -4,4 +4,9 @@ import Component from '@ember/component';
 @classic
 export default class Tables extends Component {
 
+    didRender() {
+        if (this.rendered) {
+            this.rendered();
+        }
```

Comment:
```suggestion
        this.rendered?.();
```

Reply:
I am not very familiar with ?. operator yet.
Thank you.